tests/Tests/500__jupyterhub/test-jupyterlab-git.robot

```diff
@@ -0,0 +1,182 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Test Teardown       Clean Up Server
+Suite Setup         Server Setup
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+${REPO_URL}         https://github.com/ods-qe-test/ODS-QE-Github-Test.git
+${DIR_NAME}         ODS-QE-Github-Test
+${FILE_PATH}        ODS-QE-Github-Test/test-file.ipynb
+${COMMIT_MSG}       commit msg2
+
+
+*** Test Cases ***
+Verify Pushing Project Changes Remote Repository
+    [Documentation]    Verifies that changes has been pushed successfully to remote repository
+    [Tags]    ODS-326
+    ...       Sanity    Tier1
+    Set Staging Status
+    ${randnum}=    Generate Random String    9    [NUMBERS]
+    ${commit_message}=    Catenate    ${COMMIT_MSG}    ${randnum}
+    Run Keyword And Return Status    Open New Notebook
+    Push Some Changes to Repo
```

Comment:
I think to address @bdattoma commentabout hidding the sensitive variables you have to call `RHOSi Setup` in the Suite Setup   and add  `Push Some Changes to Repo` to the list of protected keywords in RHOSi.resource

@bdattoma is that right?

Reply:
He added the keyword to `@{PROTECTED_KEYWORDS}`, is that not enough?

Reply:
@lugi0 it might be enough since the RHOSi setup in our runs (smoke, sanity, etc) will be for sure called by other suites. But, if you run only this suite the RHOSi setup wouldn't be run and the kw wouldn't be protected. 

However, I see that the kewyord `Server Setup` has the call to `RHOSi setup`. so that's ok

Reply:
Sounds good, then can we call this done?